account_fiscal_position_autodetect_optional_vies/tests/test_account_fiscal_position_vies.py

```diff
@@ -11,6 +11,8 @@ class TestAccountFiscalPostitionVies(common.SavepointCase):
     def setUpClass(cls):
         super().setUpClass()
         cls.company = cls.env.ref("base.main_company")
+        # We make sure that there is no previous record
+        cls.env["account.fiscal.position"].search([]).unlink()
```

Comment:
More than this, that can be forbidden because it might be used in other document, deactivate the auto-detect check:
```suggestion
        cls.env["account.fiscal.position"].search([("auto_apply", "=", True)]).write({"auto_apply": False})
```